Lib/test/datetimetester.py

```diff
@@ -1350,6 +1350,17 @@ def test_strftime(self):
         #check that this standard extension works
         t.strftime("%f")
 
+        # bpo-35066: make sure trailing '%' don't cause
+        # datetime's strftime to complain
+        failed = False
+        try:
+            t.strftime('%')
```

Comment:
```suggestion
            _time.strftime('%')
```

I believe the way you currently have it would not fail before your modifications.  What you've done is basically a long-winded version of:

```python
try:
    self.assertEqual(t.strftime('%'), '%')
    self.assertEqual(t.strftime('m:%m d:%d y:%y %'), 'm:03 d:02 y:05 %')
except ValueError:
    pass
```

What you *actually* want to check is that `datetime.datetime.strftime` doesn't fail *if trailing % signs are supported in `time.strftime`*.

Reply:
Good catch, I intended to call the `time` module, not use the date object's wrapper method.